doc/man3/OCSP_resp_find_status.pod

```diff
@@ -101,10 +101,10 @@ OCSP_resp_get0_id() gets the responder id of I<bs>. If the responder ID is
 a name then <*pname> is set to the name and I<*pid> is set to NULL. If the
 responder ID is by key ID then I<*pid> is set to the key ID and I<*pname>
 is set to NULL.
-OCSP_resp_get1_id() leaves ownership of I<*pid> and I<*pname> with the caller,
-who is responsible for freeing them.
-Both functions return 1 in case of success and 0 in case of failure.
-If OCSP_resp_get1_id() returns 0, no freeing of the results is necessary.
+
+OCSP_resp_get1_id() is the same as OCSP_resp_get0_id()
```

Comment:
Even better :)

Reply:
Thanks for your good comments - handled all of them.